python/generated/docstrings.h

```diff
@@ -75,6 +75,8 @@ Note: This does not match dynamics_params datasets)doc";
 
 static const char *__doc_bbp_sonata_NodePopulation_matchAttributeValues_2 = R"doc(Like matchAttributeValues, but for vectors of values to match)doc";
 
+static const char *__doc_bbp_sonata_NodePopulation_regexMatch = R"doc()doc";
```

Comment:
No documentation?

Also, should we explicitly mark these as extensions to the SONATA™ spec? Or is there an upstream PR amending this situation already?

Reply:
thanks, need to add the doc...I will do that.

I think it's worth doing a PR upstream to the docs, at least so ppl can read about the feature